packages/app/src/device/strategies/device-strategy.class.ts

```diff
@@ -19,9 +19,6 @@ export interface DeviceStrategy {
   connect(): Promise<RequestResponse<GetDeviceInfoResponseBody>>
   // AUTO DISABLED - fix me if you like :)
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  disconnect(): Promise<boolean>
```

Comment:
I'm asking to be sure. This method was dead code? How does disconnection work now?

Reply:
When you unplugged the device MC will clean up all device information (main and renderer). If you click the "disconnect" button only the renderer will forget device information.